docs/docs/ci-cd-automation/overview.md

```diff
@@ -8,12 +8,23 @@ You can find guides for:
 - [Testkube](./testkube-pipeline)
 - [Tekton](./tekton-pipeline)
 
+:::note
+If you want to see more examples with other CI/CD tools, let us know by [opening an issue in GitHub](https://github.com/kubeshop/tracetest/issues/new/choose)!
+:::
+
 Tracetest is designed to work with all CI/CD platforms and automation tools. To enable Tracetest to run in CI/CD environments, make sure to [install the Tracetest CLI](../getting-started/installation.mdx) and configure it to access your [Tracetest server](../configuration/server.md).
 
+You can also directly execute the Tracetest CLI from a docker image rather than installing the CLI on your local machine. This is sometimes convienent when you wish to execute the CLI in a CI/CD environment.
```

Comment:
```suggestion
You can also directly execute the Tracetest CLI from a Docker image rather than installing the CLI on your local machine. This is sometimes convenient when you wish to execute the CLI in a CI/CD environment.
```